### ISSUE TITLE
The published maven sources.jar is not recognized by IDEA 

### ISSUE THREAD

**Issue by lwr** [JetBrains/java-annotations#114]
now published source.jar layout is changed to
```
unzip -l /path/to/.m2/repository/org/jetbrains/annotations/26.0.0/annotations-26.0.0-sources.jar
Archive:  /path/to/.m2/repository/org/jetbrains/annotations/26.0.0/annotations-26.0.0-sources.jar
  Length      Date    Time    Name
---------  ---------- -----   ----
        0  02-01-1980 00:00   META-INF/
       25  02-01-1980 00:00   META-INF/MANIFEST.MF
        0  02-01-1980 00:00   jvmMain/
        0  02-01-1980 00:00   jvmMain/org/
        0  02-01-1980 00:00   jvmMain/org/intellij/
        0  02-01-1980 00:00   jvmMain/org/intellij/lang/
        0  02-01-1980 00:00   jvmMain/org/intellij/lang/annotations/
     6170  02-01-1980 00:00   jvmMain/org/intellij/lang/annotations/Flow.java
      744  02-01-1980 00:00   jvmMain/org/intellij/lang/annotations/Identifier.java
     4073  02-01-1980 00:00   jvmMain/org/intellij/lang/annotations/JdkConstants.java
     2948  02-01-1980 00:00   jvmMain/org/intellij/lang/annotations/Language.java
     5676  02-01-1980 00:00   jvmMain/org/intellij/lang/annotations/MagicConstant.java
     2195  02-01-1980 00:00   jvmMain/org/intellij/lang/annotations/Pattern.java
     2474  02-01-1980 00:00   jvmMain/org/intellij/lang/annotations/PrintFormat.java
     1798  02-01-1980 00:00   jvmMain/org/intellij/lang/annotations/RegExp.java
     1600  02-01-1980 00:00   jvmMain/org/intellij/lang/annotations/Subst.java
... ...
        0  02-01-1980 00:00   commonMain/
        0  02-01-1980 00:00   commonMain/org/
        0  02-01-1980 00:00   commonMain/org/intellij/
        0  02-01-1980 00:00   commonMain/org/intellij/lang/
        0  02-01-1980 00:00   commonMain/org/intellij/lang/annotations/
      796  02-01-1980 00:00   commonMain/org/intellij/lang/annotations/Identifier.kt
     2968  02-01-1980 00:00   commonMain/org/intellij/lang/annotations/Language.kt
     2233  02-01-1980 00:00   commonMain/org/intellij/lang/annotations/Pattern.kt
     1835  02-01-1980 00:00   commonMain/org/intellij/lang/annotations/RegExp.kt
     1710  02-01-1980 00:00   commonMain/org/intellij/lang/annotations/Subst.kt
... ...
---------                     -------
   130692                     69 files
```

and did not recognized by IDEA (which version is 2024.2.3 Build #IU-242.23339.11) now

**Comment by amaembo** [comment]
Thank you. Should be fixed by #115

**Comment by amaembo** [comment]
26.0.1 is released, please check
